### PR TITLE
Ed review updates

### PR DESCRIPTION
**Pull request by stuartjennings-up**
https://trello.com/c/SszxzE3c/216-3-editorial-review-updates-part-2

### CODE REVIEW

**Review by fradeve — APPROVED** [review]

**Comment by fradeve** [comment]
Can we please squash this as much as we can?

File: src/templates/core/proposals/proposal_review_block.html [hunk 1]
```diff
@@ -11,89 +11,98 @@
...
-      <a href="{% url 'withdraw_proposal_review' proposal.id reviewer.pk %}"
-         class=" btn   {% if reviewer.withdrawn %}btn-info{% else %}btn-warning{% endif %} btn-sm pull-left"
+      <a href="{% url 'withdraw_proposal_review' proposal.id review.pk %}"
+         class=" btn
+                {% if review.withdrawn %}
+                    btn-info
+                {% else %}
+                    btn-warning
+                {% endif %}
+                    btn-sm pull-left"
```
Comment (terrasea2) [inline src/templates/core/proposals/proposal_review_block.html]: Not going to ask you to change it now, but having this indented one under `{% endif %}` is a little off putting

File: src/templates/core/proposals/view_proposal.html [hunk 2]
```diff
@@ -5,161 +5,215 @@
...
+                <td>{{ proposal.subtitle }}</td>
+                <td>{{ proposal.author }}</td>
+                <td>
+                    <a href="{% url 'view_profile_readonly' proposal.owner.pk %}">
+                        {{ proposal.owner.username }}
+                    </a>
+                    <sup>
+                        <a href="core/email.html"
+                           onclick="return popitup(
+                                   '/email/proposal/{{ proposal.id }}/user/{{ proposal.owner.id }}/',
```
Comment (terrasea2) [inline src/templates/core/proposals/view_proposal.html:89]: Can't we use the `{% url blah %}` tag for this?

**Review by terrasea2 — CHANGES_REQUESTED** [review]

File: src/templates/core/proposals/view_proposal.html (hunk 2 above)
Comment (stuartjennings-up) [reply src/templates/core/proposals/view_proposal.html:89]: Yeah we probably can but this was just me reformatting existing code so didn't want to open up too much of a can of worms. This same pattern is repeated throughout the codebase.

Comment (terrasea2) [reply src/templates/core/proposals/view_proposal.html:89]: That's a maintenance nightmare waiting to happen, but I won't insist you do it. Someone will have to fix this at some stage, as I, for one, don't like this practice.

**Review by terrasea2 — APPROVED** [review]

**Pull request merged**